cor_coxph/report_sub.Rmd

```diff
@@ -66,7 +66,7 @@ paste0('cutpoints_D', Day, "bindRBD", '_', study_name))`},
 \end{table}
 ```
 
-```{r, fig.cap = 'Forest plots of hazard ratios among baseline seronegative vaccine recipients and subgroups with 95\\% point-wise confidence intervals.'}
+```{r, fig.cap = 'Forest plots of hazard ratios among baseline seronegative vaccine recipients and subgroups with 95\\% point-wise confidence intervals.',out.width = "100%"}
```

Comment:
This just overrides (at the level of this chunk) the same setting in `_common.R` (https://github.com/CoVPN/correlates_reporting/blob/master/_common.R#L108). Should we just turn it up there globally?

Reply:
Let's not touch it for now, since `immumo_graphical` and `cor_graphical` are using non-vector format graphics and so things might look bad if those images are further scaled.